docker/default.conf

```diff
@@ -3,52 +3,8 @@ agents {
     enabled = true
     port = 35000
     dispatchers {
```

Comment:
From @worldtiki:

> if these are disabled does it make sense to completely remove these blocks?
if you need to enable one it will force you to declare all of these properties (ie, we lose the defaults).

Source: https://github.com/ExpediaDotCom/haystack-agent/pull/83/files/fb3e61f9025f076460a5bdf1347e151a162264d8#r373390222

Reply:
Ignore that comment please :)

I was not aware that the defaults were baked in the code for each of the dispatchers.